microsoft/testsuites/performance/networkperf.py

```diff
@@ -286,6 +286,38 @@ def perf_udp_iperf_sriov(self, result: TestResult) -> None:
             udp_mode=True,
         )
 
+    @TestCaseMetadata(
+        description="""
+        This test case uses ntttcp to test IPv6 SR-IOV network throughput.
+        """,
+        priority=3,
+        timeout=TIMEOUT,
+        requirement=simple_requirement(
+            min_count=2,
+            network_interface=Sriov(),
+            supported_features=[IPv6],
+            unsupported_os=[BSD, Windows],
+        ),
+    )
+    def perf_tcp_ipv6_ntttcp_sriov(self, result: TestResult) -> None:
```

Comment:
Is there a specific reason to add perf test cases for IPv6? The IP version shouldn't have a significant performance impact.

Reply:
LISA currently do not have coverage for any IPv6 scenario. Hence, I thought performance is a good place to start with